<launch>

<node name="tactDriver" pkg="vizzy_tactile" type="serial_node.py" output="screen" args="/dev/ttyUSB1">
<param name="baud" type="str" value="115200" />
</node>

<node name="tactComputer" pkg="vizzy_tactile" type="tactileComputationalNode" output="screen">
</node>

</launch>
